utilities/utility.py

```diff
@@ -1,7 +1,18 @@
 import datetime
+import validators
 from helpers.database import db_session
 
 
+def validate_url(**kwargs):
+    """
+    Function to validate empty fields when
+    saving an object
+    :params kwargs
```

Comment:
Nice work @michael-basweti.
But the documentation for this function is not right yet. It is referring to empty fields instead of image url. 
Also consider changing the function name to `validate_image_url_field`. This may be more clearer and descriptive.

Reply:
Will change that right away, thanks

Reply:
I just realised that you are referring to the wrong function, that was there before

Reply:
@kampkelly , the validation can be used to validate other urls so I think it should not be named after a specific field.

Reply:
Okay. Got it.